pkg/pillar/cmd/vaultmgr/vaultmgr.go

```diff
@@ -260,18 +260,27 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject, ar
 
 	// initialize publishing handles
 	initializeSelfPublishHandles(ps, &ctx)
-	if etpm.IsTpmEnabled() {
+	tpmEnabled := etpm.IsTpmEnabled()
+	if tpmEnabled {
 		// TPM is enabled. Check if defaultVault directory exists, if not set vaultconfig
 		tpmKeyOnlyMode := checkAndPublishVaultConfig(&ctx)
 		handler.SetHandlerOptions(vault.HandlerOptions{TpmKeyOnlyMode: tpmKeyOnlyMode})
 	}
+
+	if tpmEnabled {
```

Comment:
what about integrate these messages to the if above, so we cannot have another redundant check here....

Reply:
I didn't get rid of it intentionally because these are two separate tasks, one is configuring and the other is setup,  I thought there might be added some intermediary stages in between in the two in future, and having the "SetupDefaultVault" log in the configuration part and then having the log for intermediary stage after it (which should go before setup) might cause confusion when reading logs.

what do you think?

Reply:
ok, I got it, thanks, just let as it is...